pointgrey_camera_driver/launch/camera.launch

```diff
@@ -26,5 +32,6 @@
     <node pkg="nodelet" type="nodelet" name="image_proc_debayer"
           args="load image_proc/debayer camera_nodelet_manager">
     </node>
+
```

Comment:
Thanks, I reverted all changes to this file. So by default the driver will select raw8 and the user can either change it from rqt_reconfigure or add this line to the launch file: <param name="format7_color_coding" value="rgb8" />
If you want I can add this as a note in the camera.launch file. Please let me know :)